### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #975 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#456]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 7.96 KB...]
Running smartcity.accessibility.socialnetwork.HelpfulnessTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 sec - in smartcity.accessibility.socialnetwork.HelpfulnessTest
Running smartcity.accessibility.socialnetwork.UserFunctionalityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.023 sec - in smartcity.accessibility.socialnetwork.UserFunctionalityTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.UserPinTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.032 sec - in smartcity.accessibility.socialnetwork.UserPinTest
Running smartcity.accessibility.socialnetwork.DeleteReviewsTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.046 sec - in smartcity.accessibility.socialnetwork.DeleteReviewsTest
Running smartcity.accessibility.socialnetwork.ScoreTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.725 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.database.UserManagerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.275 sec - in smartcity.accessibility.database.UserManagerTest
Running smartcity.accessibility.database.LocationManagerTest
Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.828 sec - in smartcity.accessibility.database.LocationManagerTest
Running smartcity.accessibility.database.ReviewManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.263 sec - in smartcity.accessibility.database.ReviewManagerTest
Running smartcity.accessibility.database.UserProfileManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.248 sec - in smartcity.accessibility.database.UserProfileManagerTest
Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.388 sec - in smartcity.accessibility.search.SearchTest

Results :

Tests in error: 
  ProfileImgServiceTest.UpdateProfileImageSuccess:80 » FileNotFound res/profileI...
  ProfileImgServiceTest.getProfileImageDefault:71 » IIO Can't read input file!

Tests run: 83, Failures: 0, Errors: 2, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:19 min
[INFO] Finished at: 2017-06-22T22:17:53+00:00
[INFO] Final Memory: 34M/215M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] 430a08843b4cb550f74636f06f147a985e1ee329 - change format to jpg #424

- [yaelAmitay] 8747f821c97818cd98dbef0da03ba343279622d6 - display image little change #424

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/975/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 7.96 KB...]
Running smartcity.accessibility.socialnetwork.HelpfulnessTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.008 sec - in smartcity.accessibility.socialnetwork.HelpfulnessTest
Running smartcity.accessibility.socialnetwork.UserFunctionalityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.013 sec - in smartcity.accessibility.socialnetwork.UserFunctionalityTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.UserPinTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 sec - in smartcity.accessibility.socialnetwork.UserPinTest
Running smartcity.accessibility.socialnetwork.DeleteReviewsTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.059 sec - in smartcity.accessibility.socialnetwork.DeleteReviewsTest
Running smartcity.accessibility.socialnetwork.ScoreTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.737 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.database.UserManagerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.215 sec - in smartcity.accessibility.database.UserManagerTest
Running smartcity.accessibility.database.LocationManagerTest
Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.583 sec - in smartcity.accessibility.database.LocationManagerTest
Running smartcity.accessibility.database.ReviewManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.26 sec - in smartcity.accessibility.database.ReviewManagerTest
Running smartcity.accessibility.database.UserProfileManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.204 sec - in smartcity.accessibility.database.UserProfileManagerTest
Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.458 sec - in smartcity.accessibility.search.SearchTest

Results :

Tests in error: 
  ProfileImgServiceTest.UpdateProfileImageSuccess:80 » FileNotFound res/profileI...
  ProfileImgServiceTest.getProfileImageDefault:71 » IIO Can't read input file!

Tests run: 83, Failures: 0, Errors: 2, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:17 min
[INFO] Finished at: 2017-06-22T22:22:08+00:00
[INFO] Final Memory: 34M/203M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] 430a08843b4cb550f74636f06f147a985e1ee329 - change format to jpg #424

- [yaelAmitay] 8747f821c97818cd98dbef0da03ba343279622d6 - display image little change #424

- [yaelAmitay] b171ec3e40a9605a0828fb462b5e01e24fb0fde9 - display image in the user page too #424



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/976/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 7.96 KB...]
Running smartcity.accessibility.socialnetwork.HelpfulnessTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.008 sec - in smartcity.accessibility.socialnetwork.HelpfulnessTest
Running smartcity.accessibility.socialnetwork.UserFunctionalityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.022 sec - in smartcity.accessibility.socialnetwork.UserFunctionalityTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.UserPinTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.043 sec - in smartcity.accessibility.socialnetwork.UserPinTest
Running smartcity.accessibility.socialnetwork.DeleteReviewsTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.067 sec - in smartcity.accessibility.socialnetwork.DeleteReviewsTest
Running smartcity.accessibility.socialnetwork.ScoreTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.689 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.database.UserManagerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.09 sec - in smartcity.accessibility.database.UserManagerTest
Running smartcity.accessibility.database.LocationManagerTest
Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.354 sec - in smartcity.accessibility.database.LocationManagerTest
Running smartcity.accessibility.database.ReviewManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.221 sec - in smartcity.accessibility.database.ReviewManagerTest
Running smartcity.accessibility.database.UserProfileManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.186 sec - in smartcity.accessibility.database.UserProfileManagerTest
Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.374 sec - in smartcity.accessibility.search.SearchTest

Results :

Tests in error: 
  ProfileImgServiceTest.UpdateProfileImageSuccess:80 » FileNotFound res/profileI...
  ProfileImgServiceTest.getProfileImageDefault:71 » IIO Can't read input file!

Tests run: 83, Failures: 0, Errors: 2, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:12 min
[INFO] Finished at: 2017-06-22T22:26:57+00:00
[INFO] Final Memory: 34M/203M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] 430a08843b4cb550f74636f06f147a985e1ee329 - change format to jpg #424

- [yaelAmitay] 8747f821c97818cd98dbef0da03ba343279622d6 - display image little change #424

- [yaelAmitay] b171ec3e40a9605a0828fb462b5e01e24fb0fde9 - display image in the user page too #424

- [yaelAmitay] 6d7209a95c76ffc36c466760a2212b0bf9799691 - forgot new file and import in client side



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/977/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
